apps/admin-web/src/services/apollo/index.ts

```diff
@@ -0,0 +1,74 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+import { BatchHttpLink } from '@apollo/client/link/batch-http'
+import { setContext } from '@apollo/client/link/context'
+
+import { ENVIRONMENT } from '@admin-web/env'
+import { apiUrl } from '@admin-web/services/httpClient'
+// import { uniqBy } from '@admin-web/utils/uniqBy'
```

Comment:
I don't know if it is neccessary in the future, if that time comes I could easily uncomment it. Should I remove it anyways?

Reply:
i don't think u really need cache for backoffice imo. It's there to cache courses response from `search` query used in cugetreg search bar, so that search queries with the same keyword and filters can use the same cached response. In our backoffice use case, we don't fetch many times in one session, making the cache not really useful.